script.program.fritzact/addon.xml

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<addon id="script.program.fritzact" name="Fritz Smart Home" version="0.0.17" provider-name="Birger Jesch">
+    <requires>
+        <import addon="xbmc.python" version="2.24.0" />
+        <import addon="script.module.requests" version="2.9.1" />
+    </requires>
+	<extension point="xbmc.python.script" library="default.py">
```

Comment:
Just a tip, the smaller you can get `default.py` to be, the faster your addon will start up

Reply:
Is this a requirement or just a tip? I do not want to complain, but I've been waiting for a month now. And I have other projects within the pipeline too.

Reply:
Just a tip,  not blocking. I think nobody had any time to review this yet. :(